client/src/App.tsx

```diff
@@ -22,10 +22,11 @@ const App = () => {
   return (
     <ChakraProvider>
       <div id="app">
-        <Example />
+        <HomePage />
+        {/* <Example />
         {
           user ?  <HomePage /> : <LoginPage setUser={setUser}/>
```

Comment:
can you make sure to leave this uncommented? We want the user to have to sign in and save the state to redux before they are taken to the HomePage compnent